inginious/frontend/pages/course_admin/statistics.py

```diff
@@ -14,6 +14,56 @@ def compute_statistics(task, data):
         [(<tag>, '61%', '50%'), (<tag>, '76%', '100%'), …]
     )
      """
+    
+    super_dict = {}
+    for submission in data:
+        task = tasks[submission["taskid"]]
+        username = "".join(submission["username"])
+        tags_of_task = task.get_tags()[0] + task.get_tags()[1]
+        for tag in tags_of_task:
+            if tag not in super_dict:
```

Comment:
I think that the ``setdefault`` method can be useful here to make some clean up.